aws/templates/common.ftl

```diff
@@ -1943,12 +1957,12 @@ behaviour.
     [#return networkEndpoints]
 [/#function]
 
-[#function getDeploymentProfile typeObject deploymentMode ]
+[#function getDeploymentProfile occurrenceProfiles deploymentMode ]
 
     [#local deploymentProfileNames = []]
 
-    [#if (typeObject.Profiles.Deployment)?? ]
-        [#list asArray(typeObject.Profiles.Deployment) as profileName ]
+    [#if (occurrenceProfiles)?? ]
```

Comment:
Compiler won't let you call getDeploymenrProfile without occurrenceProfiles.

I think the intent is it must be there so just drop the if test.

Also the other asArrays could be modified with a missing default of ![] to remove the if test - I think it looks cleaner